modules/mini/src/main/java/org/apache/fluo/mini/MiniFluoImpl.java

```diff
@@ -214,7 +214,7 @@ public static void main(String[] args) {
         System.exit(-1);
       }
       try (MiniFluo mini = new MiniFluoImpl(config)) {
-        log.info("MiniFluo is running");
+        log.info("MiniFluo is running " + mini);
```

Comment:
Why did you add this to the log message?  I don't think MiniFluoImpl has a `toString()` method.  I am not sure anything calls this main method since the mini-fluo script was removed in  6fd8629.  We may be able to remove it.